compiler/testData/codegen/bytecodeText/kt10259.kt

```diff
@@ -28,7 +27,8 @@ inline fun test(s: () -> Unit) {
 // this behavior is equivalent to javac and seems to be correct.
 
 // JVM_IR_TEMPLATES
-// 2 INNERCLASS Kt10259Kt\$box\$1\$1\s
+// 5 INNERCLASS
```

Comment:
Am I correct that old backend misses one entry? Or it opposite situation?

Reply:
Yes, the old backed regenerates the class and to be 100% correct the regenerated copy should be added to inner classes of Kt10359Kt.